src/green/Green.jl

```diff
@@ -97,15 +97,15 @@ mutable struct Green2DLR{T<:Number,Type<:TimeDomain,TGT,SGT}
     """
     function Green2DLR{T}(name::Symbol, timeType::TT, β, isFermi::Bool, Euv, spaceGrid, color::Int = 1;
         timeSymmetry::Symbol = :none, rtol = 1e-8, kwargs...
-    ) where {T<:Number, TT<:TimeDomain}
+    ) where {T<:Number,TT<:TimeDomain}
         # @assert spaceType == :k || spaceType == :x
         @assert timeSymmetry == :ph || timeSymmetry == :pha || timeSymmetry == :none
 
         spaceType = :k #TODO: replace it with spaceGrid.type after we have a spaceGrid package
 
         dlrGrid = DLRGrid(Euv, β, rtol, isFermi, timeSymmetry)
 
-        println(keys(kwargs))
+        # println(keys(kwargs))
```

Comment:
there's no meaningful change except here in this file, but the space changes will introduce conflicts when merging with the other pull request. I would suggest revert changes in this file.